Mathlib/Order/Synonym.lean

```diff
@@ -50,8 +50,8 @@ variable {α : Type*}
 
 namespace OrderDual
 
-instance [h : Nontrivial α] : Nontrivial αᵒᵈ :=
-  h
+instance [h : Nontrivial α] : Nontrivial αᵒᵈ := h
+instance [h : Unique α] : Unique αᵒᵈ := h
```

Comment:
Can you make sure all other type synonyms in this file have this instance?

Reply:
There's some other missing instances here. I've put them in #32242 so as to not clog this PR.